ArdusatSDK.cpp

```diff
@@ -10,13 +10,11 @@
 
 #include "ArdusatSDK.h"
 
-RTC_DS1307 RTC;
-
-SdVolume vol;
-#define _output_buffer vol.cacheAddress()->output_buf
-int _output_buf_len = 0;
-
+int OUTPUT_BUF_SIZE = 256;
+int OUTPUT_BUFFER_MAXSIZE = 250;
 bool ARDUSAT_SHIELD = false;
+char * _output_buffer;
+int _output_buf_len = 0;
```

Comment:
make these `static` - they're internal "implementation" variables for this particular source file